modules/swagger-parser/src/main/java/io/swagger/parser/processors/DefinitionsProcessor.java

```diff
@@ -29,7 +29,7 @@ public void processDefinitions() {
             return;
         }
 
-        Set<String> keySet = new HashSet<>();
+        Set<String> keySet = new LinkedHashSet<>();
```

Comment:
As with before, not clear why we care about order for these internal storage mechanisms.